docker/musicbrainz-website/consul-template-template-renderer.conf

```diff
@@ -7,7 +7,7 @@ template {
 exec {
     command = "start_template_renderer.sh"
     splay = "10s"
-    reload_signal = "SIGHUP"
+    reload_signal = "SIGTERM"
```

Comment:
So, part of the necessary changes is that the template renderer is now built with Webpack. Everything it needs is bundled into a single file, including itself and all of the React components it can render (those have to be specified in root/server/components.js, so that Webpack can pick them up).

`SIGHUP` only spawns new workers. That's not really useful if the configuration (DBDefs) changes, because the configuration is compiled *into the bundle* that is executing. We have to give a `SIGTERM` now to rebundle the whole thing.

Reply:
This comment is no longer accurate, since I changed it to only bundle root/server/components.js and not the entire server. This is a more conservative change with less chance of causing deployment issues. (But it means we can't use ES modules or non-comment-Flow-types in `root/server*` or files required through there.)